docs/es-overview.asciidoc

```diff
@@ -2,169 +2,38 @@
-image::images/workflow.png[Elastic Security workflow]
-
-Here's an overview of the flow and its components:
-
-* Data is shipped from your hosts to {es} in the following ways:
-** <<install-endpoint, {elastic-defend}>>: {agent} integration that
-protects your hosts <<malware-prevention, against malware>> and ships these data sets:
-***  *Windows*: Process, network, file, DNS, registry, DLL and driver loads,
-malware security detections, API
-*** *Linux/macOS*: Process, network, file
-** {integrations-docs}[{integrations}]: Integrations are a streamlined way to send your data to the {stack}. Integrations are available for popular services and platforms, like Nginx, AWS, and MongoDB, as well as many generic input types like log files.
-** https://www.elastic.co/integrations?solution=security[Beat modules]: {beats}
-are lightweight data shippers. Beat modules provide a way of collecting and
-parsing specific data sets from common sources, such as cloud and OS events,
-logs, and metrics. Common security-related modules are listed
-<<enable-beat-modules, here>>.
-* The {security-app} in {kib} is used to manage the *Detection engine*,
-*Cases*, and *Timeline*, as well as administer hosts running {elastic-defend}:
-** Detection engine: Automatically searches for suspicious host and network
-activity via the following:
-*** <<detection-engine-overview, Detection rules>>: Periodically search the data
-({es} indices) sent from your hosts for suspicious events. When a suspicious
-event is discovered, an alert is generated. External systems, such as
-Slack and email, can be used to send notifications when alerts are generated.
-You can create your own rules and make use of our <<prebuilt-rules, prebuilt ones>>.
-*** <<detections-ui-exceptions, Exceptions>>: Reduce noise and the number of
-false positives. Exceptions are associated with rules and prevent alerts when
-an exception's conditions are met. *Value lists* contain source event
-values that can be used as part of an exception's conditions. When
-{elastic-defend} is installed on your hosts, you can add malware exceptions
-directly to the endpoint from the Security app.
-*** <<included-jobs, {ml-cap} jobs>>: Automatic anomaly detection of host and network events. Anomaly scores are provided per host and can be used with detection rules.
-** <<timelines-ui, Timeline>>: Workspace for investigating alerts and events.
-Timelines use queries and filters to drill down into events related to
-a specific incident. Timeline templates are attached to rules and use predefined
-queries when alerts are investigated. Timelines can be saved and shared with
-others, as well as attached to Cases.
-** <<cases-overview, Cases>>: An internal system for opening, tracking, and sharing
-security issues directly in the Security app. Cases can be integrated with
-external ticketing systems.
-** <<admin-page-ov, Administration>>: View and manage hosts running {elastic-defend}.
+[[siem-integration]]
+=== Learn more
```

Comment:
I think we can break this section down even more to organize the information and create a visual guide for the page. 

When I look at the content of this section, I see three major topics: 
- Tips and docs for getting an overview of Elastic Security
- Docs for popular Security features, including Elastic Endpoint
- Docs for ES and Kibana, which provide foundational functionality for Security 

I think the first two topics could have their own sections. Maybe something like "Try Elastic Security" and "Learn about core features". The third topic is a bit harder to loop in. It's related but is more important to understand when you're ingesting data and working with Kibana apps. Maybe we can summarize lines [27](https://github.com/elastic/security-docs/pull/4528/files#diff-0e04bc5bc0b046b3e7fa6c086e42a070be04d5e45f18b5a8fa43bd11019d1dc7R27)-[33](https://github.com/elastic/security-docs/pull/4528/files#diff-0e04bc5bc0b046b3e7fa6c086e42a070be04d5e45f18b5a8fa43bd11019d1dc7R33) and fold that info into the intro para for the page? [Here's](https://www.elastic.co/guide/en/kibana/current/kibana-concepts-analysts.html) an example of what I'm thinking about.

Reply:
@nastasha-solomon I agree with reorganizing this section, but I see it in a slightly different direction. The intro para could be streamlined into a single bullet: I don't think we need to explain how to use the docs' side menu, and we don't even need a lead-in between the heading and list:

> ### Learn about Elastic Security
> * Get started - System requirements, workspaces, configuration, and data ingestion.
> * Elastic Security UI overview - Navigate Elastic Security's tools and interfaces.
> * Etc, etc.

That seems pretty clear without needing extra verbiage. 

But I agree that the ES & Kibana stuff is a separate thought and could stand as a separate section.